packages/proxyIdentity/migrations/2_identityManager_migrations.js

```diff
@@ -2,10 +2,13 @@
 const { deployProxy } = require('@openzeppelin/truffle-upgrades');
 
 // needs to be updated with actual address
-const libraryAddress = '0x4B0897b0513fdC7C541B6d9D7E929C4e5364D2dB';
+const libraryAddress = '0x4c392f00a17B1b77aB059E6dc9ccE91F74113c66';
```

Comment:
@nichonien is this comment of `// needs to be updated with actual address` still accurate? What contract is `'0x4c392f00a17B1b77aB059E6dc9ccE91F74113c66'`?

Reply:
That is correct address for the latest deployment. This comment is from the standpoint when someone else uses this script, this address would be irrelevant at that point of time for a fresh deployment. Maybe I should rephrase this comment?

Reply:
Updated the comment.